### ISSUE TITLE
Pyfa won't start after update from v1.x

### ISSUE THREAD

**Issue by DaddyGnome** [pyfa-org/Pyfa#1603]
## Bug Report
Pyfa won't start

### Expected behavior:
Pyfa starts

### Actual behavior:
Error window with following:

OS version: Windows-10-10.0.17134-SP0
Python version: 3.6.1 (v3.6.1:69c0db5, Mar 21 2017, 17:54:52) [MSC v.1900 32 bit (Intel)]
wxPython version: 4.0.0b2 (wxWidgets 3.0.4)
SQLAlchemy version: 1.1.10
Logbook version: 1.0.0
Requests version: 2.18.4
Dateutil version: 2.6.0

####################

Traceback (most recent call last):
  File "C:\Users\holme\Documents\Sync\Git\blitzmann\Pyfa\pyfa.py", line 135, in <module>
  File "C:\Users\holme\Documents\Sync\Git\blitzmann\Pyfa\gui\mainFrame.py", line 167, in __init__
  File "C:\Users\holme\Documents\Sync\Git\blitzmann\Pyfa\gui\bitmap_loader.py", line 65, in getBitmap
  File "C:\Users\holme\Documents\Sync\Git\blitzmann\Pyfa\gui\bitmap_loader.py", line 87, in getImage
TypeError: initial_value must be str or None, not bytes

### Detailed steps to reproduce:
Upgrade from fairly recent 1.3x version to v2.02. Start pyfa.

### Fits involved in EFT format (Edit > To Clipboard > EFT):
n/a

### Release or development git branch? Please note the release version or commit hash:
Downloaded version recommended by previous version upgrade dialog.

### Operating system and version (eg: Windows 10, OS X 10.9, OS X 10.11, Ubuntu 16.10):
Win 10.

**Comment by blitzmann** [comment]
duplicate #1582, see there for fix. 

tldr: Please read release notes, can't overwrite with zip file, remove or uninstall previous installation before using 2.x.

**Comment by DaddyGnome** [comment]
For what it's worth, I did use the installer, which uninstalled the previous version first...
It did complain at the time that some elements would have to be removed manually, but I wasn't sure what they were. I have now removed all trace of pyfa from my system and reinstalled and it works.
Thanks for the help.
